### DISCUSSION TITLE
为什么发布的是二进制文件，电脑无法直接打开，我应该如何编译它，使它在电脑上能运行

### DISCUSSION THREAD

**Discussion by zhuanshiyan** [ipxe/wimboot#54] [Q&A]
为什么发布的是二进制文件，我下载下来后电脑无法运行，请问我应该如何编译它，使它在电脑上能运行

**Comment by NiKiZe** [comment]
You are free to ask questions in English.

**Reply by zhuanshiyan** [reply]
Why was a binary file released instead of an executable .exe file? I downloaded it, but my computer can't run it. How should I compile it to make it run on my computer?

**Reply by NiKiZe** [reply]
I think you need to read a how-to on what wimboot is and how to use it, it is not a windows program.

What is it that you want to use wimboot for, and which infrastructure do you have in place so far?

**Reply by zhuanshiyan** [reply]
I see, I always thought it was a Windows program，I am just a computer assembly enthusiast.
